src/components/ResultsEmpty.tsx

```diff
@@ -34,8 +39,15 @@ export const ResultsEmpty: React.FC<ResultsEmptyProp> = ({
         <Text fontWeight='bold' fontSize='lg' letterSpacing='0.02em' translation={title} />
         <Text color='text.subtle' letterSpacing='0.012em' translation={body} />
         {ctaHref && (
```

Comment:
Since you can't have one without the other

```suggestion
        {ctaHref && ctaText (
```

Then remove the `ctaText` default and pass it explicitly in `<PositionTable />` and `<WalletProviderTable />`, same like we're doing here in this PR in `<PriceTable />`